corehq/motech/repeaters/templates/repeaters/repeat_record_report.html

```diff
@@ -11,9 +11,58 @@
         <script src="{% static 'ace-builds/src-min-noconflict/mode-xml.js' %}"></script>
         <script src="{% static 'ace-builds/src-min-noconflict/ext-searchbox.js' %}"></script>
         <script src="{% static 'repeaters/js/repeat_record_report.js' %}"></script>
+        <script src="{% static 'repeaters/js/repeat_record_report_selects.js' %}"></script>
     {% endcompress %}
 {% endblock js %}
 
+{% block filter_panel %} {{ block.super }}
+  <div>
+      {% if total > 0 %}
+        <div id ="form_options" class="well form-inline" style="margin: 1em">
+          <div>
+            <button id="resendAll" class="btn btn-default">Resend selected</button>
+            <button id="cancelAll" class="btn btn-default">Cancel selected</button>
+            <button id="requeueAll" class="btn btn-default">Requeue selected</button>
+            <label class="checkbox" >
+              <input type="checkbox" id="select_all" value="{{ form_query_string|urlencode }}"
```

Comment:
It's pretty unusual to use a checkbox value other than blank/`on`. Can you explain a bit more what value is being stored here and how it's being used?